www/_includes/assets/css/_global.scss

```diff
@@ -220,3 +220,33 @@ main {
 pre {
   overflow-x: auto; // 2.
 }
+
+main table {
+  border-collapse: collapse;
+}
+
+main th {
+  border-bottom: 1px solid
+    color-mix(in oklch, #{$color-foreground}, transparent 75%);
```

Comment:
There wasn’t really a border token. Is this stupid?

Reply:
if we can't do stupid things on our own website, what's the point of having a website?